src/components/MultiHopTrade/hooks/useGetTradeQuotes/useGetTradeQuotes.tsx

```diff
@@ -124,10 +124,7 @@ export const useGetTradeQuotes = () => {
   const isDebouncing = debouncedSellAmountCryptoPrecision !== sellAmountCryptoPrecision
 
   const sellAccountId = useAppSelector(selectFirstHopSellAccountId)
-  // No need to pass a sellAssetAccountId to synchronize the buy account here - by the time this is called, we already have a valid buyAccountId
-  const buyAccountId = useAppSelector(state =>
-    selectLastHopBuyAccountId(state, { accountId: undefined }),
-  )
```

Comment:
no need to worry about this case of not passing a default, the selector handles this internally anyway